scripts/style_mechanics_normalize.py

```
@@ -0,0 +1,22 @@
+import re
+
+JP_TO_EN = str.maketrans({"、": ", ", "。": ". ", "「": '"', "」": '"', "／": "/", "　": " "})
```

Ruff (0.12.2)

String contains ambiguous `／` (FULLWIDTH SOLIDUS). Did you mean `/` (SOLIDUS)?

(RUF001)